syntax = "proto3";

message User{
  int32   id         = 1;
  string  first_name = 2;
  string  last_name  = 3;
  string  email      = 4;
  string  gender     = 5;
  string  ip_adress  = 6;
  string  user_name  = 7;
  string  agent      = 8;
  string country     = 9;

}

message UserResponse {
  bool success = 1;
}

service UserService {
  // unary rpc
  rpc UserWriteRedis(User) returns(UserResponse){};

}